src/Payment.ts

```diff
@@ -260,24 +260,28 @@ export class Payment {
   /**
    * Creates a payment request link.
    * @param networkAddress Address of a currency network.
-   * @param amount Requested transfer amount.
-   * @param subject Additional information for payment request.
-   * @param customBase Optional custom base for link. Default `trustlines://`.
+   * @param options
+   *        options.amount - optional amount for the payment request
```

Comment:
Did we define if this is in base units or "normal", so if I request 1 euro, do I but there a 1 or 100000000

Reply:
How do you use it atm?